src/app/common/services/channels/channels.service.ts

```diff
@@ -16,122 +16,162 @@ export class ChannelsService {
   ) { }
 
   addChannel(channel: Channel) {
-    return this.http.post(environment.channelsUrl, channel, { observe: 'response' }).map(
-      resp => {
-        return resp;
-      },
-      err => {
-        this.notificationsService.error('Failed to create Channel',
-          `'Error: ${err.status} - ${err.statusTexts}`);
-      },
-    );
+    return this.http.post(environment.channelsUrl, channel, { observe: 'response' })
+      .map(
+          resp => {
+            return resp;
+          },
+        )
+      .catch(
+        err => {
+          this.notificationsService.error('Failed to create Channel',
+            `'Error: ${err.status} - ${err.statusText}`);
```

Comment:
What is this `'` before `Error`? Is it needed to be `'Error` instead of just `Error`?